opentelemetry-api/src/opentelemetry/distributedcontext/__init__.py

```diff
@@ -11,3 +11,113 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from contextlib import contextmanager
+import typing
+
+PRINTABLE = set(chr(num) for num in range(32, 127))
```

Comment:
Please don't use magic numbers here. Isn't there a (combination of) constants that covers this in [`string](https://docs.python.org/3/library/string.html)` (I can't say for sure, since I can only speculate what "32" and "127" are without looking them up 😃)